app/views/layouts/_header.html.erb

```diff
@@ -0,0 +1,29 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+  <div class="container">
+    <%= link_to t(".sample app"), root_path, id: "logo" %>
+    <nav>
+      <ul class="nav navbar-nav navbar-right">
+        <li><%= link_to t(".Home"), root_path %></li>
+        <li><%= link_to t(".Help"), help_path %></li>
+        <% if logged_in? %>
+          <li><%= link_to t(".users"), "#" %></li>
+          <li class="dropdown">
+            <a href="#" class="dropdown-toggle" data-toggle="dropdown">
+              Account <b class="caret"></b>
+            </a>
+            <ul class="dropdown-menu">
+              <li><%= link_to t(".profile"), current_user %></li>
+              <li><%= link_to t(".Settings"), "#" %></li>
+              <li class="divider"></li>
+              <li>
+                <%= link_to t(".Log out"), logout_path, method: :delete %>
+              </li>
+            </ul>
+          </li>
+        <% else %>
+          <li><%= link_to t(".Log in"), login_path %></li>
```

Comment:
tên biến ko được bắt đầu bằng từ viết hoa
tên biến ko cách ra thế em nhé
`Log in"`